index.html

```diff
@@ -10,23 +10,33 @@
 
 <body class="body">
 
-  <div class="navbar">
-    <div class="my-logo">My Logo</div>
-    <a class="menu-icon"></a>
-  </div>
+  <section class="intro">
+    <header>
+      <nav class="navbar">
+        <a class="my-logo">My Logo</a>
+        <a class="menu-icon"></a>
+      </nav>
+    </header>
 
-  <div class="main">
-    <div class="primary-text">I'm Juan Diaz Glad to see you!</div>
-    <div class="secondary-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
-    <div class="connect-text">LET'S CONNECT</div>
-    <div class="socialmedia-icons">
-      <a class="twitter-icon"></a>
-      <a class="linkedin-icon"></a>
-      <a class="m-icon"></a>
-      <a class="cat-icon"></a>
-      <a class="fingers-icon"></a>
+
+    <div class="main">
```

Comment:
[Optional]

- [ ] Instead of using the `div` tag please use the `section` tag because of SEO purposes. :+1: